mitosheet/mitosheet/tests/decorators.py

```diff
@@ -35,6 +35,11 @@
     reason='This test only runs on later versions of Pandas. API inconsistencies make it fail on earlier versions'
 )
 
+pandas_post_1_5_only = pytest.mark.skipif(
```

Comment:
This was released like last week, so if we have things failing pre 1.5 we should def spend the time to make them work elsewhere!

Reply:
Just added this since we had one for all of the other pandas versions -- we're not using anything that recent though.